### PR TITLE
CLN: Removed outdated comment

### PR DESCRIPTION
**Pull request by ShaharNaveh**
- [ ] closes #xxxx
- [ ] tests added / passed
- [x] passes `black pandas`
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

I think this is a bit outdated, since we are using black and it's doing the work for us, right?

according to git blame, this is 2 years old.

### CODE REVIEW

**Review by datapythonista — APPROVED** [review]

File: pandas/tests/arrays/categorical/test_repr.py [hunk 1]
```diff
@@ -147,8 +147,6 @@ def test_categorical_repr_datetime(self):
         idx = date_range("2011-01-01 09:00", freq="H", periods=5)
         c = Categorical(idx)
 
-        # TODO(wesm): exceeding 80 characters in the console is not good
-        # behavior
```
Comment (jbrockmendel) [inline pandas/tests/arrays/categorical/test_repr.py]: is this referring to the code or to the string being returned?  black only affects the former

Comment (TomAugspurger) [reply pandas/tests/arrays/categorical/test_repr.py]: It's about the output.

```python
In [16]: idx = pd.CategoricalIndex(list(string.ascii_letters))

In [17]: [len(x) for x in repr(idx).split('\n')]
Out[17]: [77, 77, 77, 77, 38, 107]
```

**Comment by WillAyd** [comment]
Thanks @MomIsBestFriend

**Comment by alimcmaster1** [comment]
@MomIsBestFriend - I don’t think this is referring to black.

Think it’s related to https://github.com/pandas-dev/pandas/issues/22524

**Comment by MomIsBestFriend** [comment]
@alimcmaster1 should it be reverted then?

**Pull request merged**